### DISCUSSION TITLE
how can I center a div?

### DISCUSSION THREAD

**Discussion by dara-marotto** [community/community#43597] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm starting study CSS in 2023

**Comment by kelwin-hollunder** [comment, marked as answer]
You can use Flexbox for almost everything lol
Can you learn Flex here: https://developer.mozilla.org/pt-BR/docs/Web/CSS/flex
Can you pratice Flex here: https://flexboxfroggy.com/

As you dive deep more in CSS, you should know grid too.
Here you can learn grid: https://developer.mozilla.org/pt-BR/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout
Here you can pratice: https://cssgridgarden.com/

**Reply by dara-marotto** [reply]
thanks for your help!
